pythonforandroid/recipes/zope_interface/__init__.py

```diff
@@ -25,7 +25,11 @@ def build_arch(self, arch):
     def prebuild_arch(self, arch):
         super(ZopeInterfaceRecipe, self).prebuild_arch(arch)
         with current_directory(self.get_build_dir(arch.arch)):
-            sh.rm('-rf', 'src/zope/interface/tests', 'src/zope/interface/common/tests')
+            sh.rm(
+                '-rf',
+                'src/zope/interface/tests',
+                'src/zope/interface/common/tests',
+            )
```

Comment:
We shorten the line, to be below 79 characters, to force `rebuild_updated_recipes` script to be triggered.